### DISCUSSION TITLE
How fix create-cluster -o --start

### DISCUSSION THREAD

**Discussion by updategroup** [bloxbean/yaci-devkit#17] [Q&A]
I have make tutorial https://docs.google.com/document/d/1PoAqQu7NweEz8_WCQFKb44tkufjEStUpkGTj3G17eqw/edit
Failed when create-cluster -o --start
<img width="1103" alt="image" src="https://github.com/bloxbean/yaci-devkit/assets/26798127/2c1e528d-bc8c-47f8-9ba6-89e12d5ae348">

**Comment by satran004** [comment]
@updategroup Could you please comment out or remove the 'volume' section from the docker-compose.yml file? 
After that, please run the commands to stop and then start the service again.
```
    volumes:
      - ~/clusters:/clusters
```
Some users have encountered a similar issue because of filesystem permission problems, as DevKit uses a shared folder.

https://github.com/bloxbean/yaci-devkit/issues/11
